### ISSUE TITLE
Turn Marker not visible for players in Foundry 0.8.6 with token vision.

### ISSUE THREAD

**Issue by Daso78** [kckaiwei/TurnMarker-alt#70]
I'm using Foundry 0.8.6 with Pathfinder 2E (2.0.3) and Turn Marker 2.8.0

If the scene's token vision is turn ON, then the players do not see the turn marker tile. If I disable the token vision setting, the turn marker tile appears.

![Turn Marker Error 1](https://user-images.githubusercontent.com/85315021/120705077-58890000-c485-11eb-9969-076259344fea.png)
![Turn Marker Error 2](https://user-images.githubusercontent.com/85315021/120705081-59219680-c485-11eb-9cd1-f00a8e228209.png)

**Comment by Stountain** [comment]
I am having this same issue

**Comment by kckaiwei** [comment]
Ok, I'm looking into it. Was kinda caught off guard by the switch of 0.8.6 to release, and that changed how tiles are built/handled throughout Foundry, so there's a lot of new caveats/bugs for me to figure out.

**Comment by jrmiller82** [comment]
Same bug here.

**Comment by Danii0206** [comment]
Glad I'm not the only one with this. Same bug here. ^^

**Comment by kckaiwei** [comment]
Yeah, it's a bitch of a bug to figure out, since I can't find much Tile vision examples out there. If I still can't figure it out much longer, I'll probably end up switching it to using sprites instead of Tiles :\.

**Comment by chklein1** [comment]
The issue repros on 0.8.7 with DnD5e 1.3.5 as well.

**Comment by zappallot** [comment]
Same here, 0.8.8 with pf2e 2.6.0

**Comment by STBaf** [comment]
Just for information: Same problem with 0.8.8 (and ds4 as game system - but I think it's game-system independent)

**Comment by MarkPearce** [comment]
same issue, marker is only visible for the GM.
Same problem in both PF2 and SWADE rulesets.

watching for merge of bugfix  I  miss the starting square indicator the mod provides!

**Comment by ckdragons** [comment]
I am having the same issue as reported here. Using Foundry 0.8.9 with Token Marker 2.8.1.

**Comment by BlackWand1** [comment]
Hi!

Any update on this issue?

**Comment by BlackWand1** [comment]
Hi!

Looks to be working in 2.8.2 with D&D 3.5 system.
But still get some errors (not affecting anything that I can visible see in the scene though)

![image](https://user-images.githubusercontent.com/85252943/140622610-04d9f310-55e9-4d09-bf73-ed9331f8d08b.png)
